build/vue-loader.conf.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   }),
   postcss: [
     require('autoprefixer')({
-      browsers: ['last 2 versions']
+      overrideBrowserslist: ['last 2 versions']
```

Comment:
Why this change was added?

Reply:
because when I'm compiling on dev, npm suggests do this change

Reply:
Why we are compiling on dev? We should just create PR for required changes only and compiling will be done when we publish it.

Reply:
Docs files are fine as those are needed to update docs but we can skip dist ones